Proposal for the basic structure of a simulation study.md

```diff
@@ -0,0 +1,49 @@
+# Proposal for the basic structure of a simulation study
+
+### Final Structure
+
+1. *Defining aims & objectives (including hypotheses)*
+2. *Simulation procedure /Design & Data-generation mechanism*
+3. *Methods selection (including software and estimands)*
+4. *Optional: Model specification*
+5. *Selection of performance measures*
+6. *Coding & execution*
```

Comment:
I agree with this step. If not mentioned here some step should include a method to handle  non-convergent / "imperfect" simulation runs.

Reply:
I will check and include if missing